src/pages/Cart/Cart.js

```diff
@@ -0,0 +1,105 @@
+import { Link } from "react-router-dom";
+import { useCart } from "../../contexts/CartProvider";
+import CartCard from "../../components/CartCard/CartCard";
+import DefaultWithoutSearch from "../../layouts/DefaultWithoutSearch";
+import styles from "./Cart.module.css";
+import { ReactComponent as EmptyCartSvg } from "./EmptyCartImage.svg";
+import { useAuth } from "../../contexts/AuthProvider";
+
+const Cart = () => {
+  const { token } = useAuth();
+  const { cartState, isCartLoading } = useCart();
+  const deliveryCharge = cartState.totalPrice > 500 ? 50 : 0;
+
+  if (isCartLoading) {
+    return (
+      <DefaultWithoutSearch>
+        <h1 className="overlay">Loading ...</h1>
+      </DefaultWithoutSearch>
+    );
+  }
+
+  if (cartState.items.length <= 0) {
+    return (
+      <DefaultWithoutSearch>
+        <div className={styles.emptyCartContainer}>
+          <EmptyCartSvg width="80%" />
+          <p
+            style={{
+              fontWeight: "bold",
+              fontSize: "1.2rem",
+              padding: "1rem 2rem",
+            }}>
+            You cart is empty!
+          </p>
+          <p>Add items to it now</p>
+          <Link to="/products" className="btn links btn-link">
+            Shop now
+          </Link>
+        </div>
+      </DefaultWithoutSearch>
+    );
+  }
+
+  return (
+    <DefaultWithoutSearch>
+      <h3 className={styles.cartHeader}>
+        My Cart ({cartState.items.length > 0 && cartState.items.length})
```

Comment:
This condition not required `cartState.items.length > 0 && cartState.items.length`.
This rertun will not execute when `cartState.items.length <= 0` due to the previous `if`